sorting/merge_two_sorted_array.cpp

```diff
@@ -0,0 +1,21 @@
+class Solution {
+public:

```

Comment:
The code looks good, could you add description with sample io on top of file? , thanks

Reply:
Yup sure